### ISSUE TITLE
Missing mail template

### ISSUE THREAD

**Issue by zce** [TryGhost/Ghost#9368]
### Issue Summary

no such file or directory, `ghost/versions/1.19.1/core/server/services/mail/templates/welcome.html`

![image](https://user-images.githubusercontent.com/6166576/34550073-d7bde002-f148-11e7-8631-82218de53082.png)

### Steps to Reproduce

1. Use `ghost-cli` install a new blog with v1.19.1
```shell
$ ghost install local
✔ Checking system Node.js version
✔ Checking current folder permissions
ℹ Checking operating system [skipped]
ℹ Checking MySQL is installed [skipped]
✔ Checking for latest Ghost version
✔ Setting up install directory
✔ Downloading and installing Ghost v1.19.1
✔ Finishing install process
✔ Configuring Ghost
✔ Setting up instance
✔ Running database migrations
✔ Validating config
✔ Starting Ghost
You can access your blog at http://localhost:2368/

Ghost uses direct mail by default
To set up an alternative email method read our docs at https://docs.ghost.org/docs/mail-config
```
2. http://localhost:2368/ghost/#/setup/two

### Technical details:

* Ghost Version: 1.19.1

**Comment by gramakri** [comment]
We hit this when updating the Cloudron package as well.

The templates directory is missing from the npm package. It's [here](https://github.com/TryGhost/Ghost/tree/master/core/server/services/mail) though.

```
root@1e03bbc5de8d:/home/cloudron/ghost/versions/1.19.1/core/server/services/mail# ls -l
total 16
-rw-r--r-- 1 cloudron cloudron 4176 Jan  3 14:39 GhostMailer.js
-rw-r--r-- 1 cloudron cloudron   84 Jan  3 14:39 index.js
-rw-r--r-- 1 cloudron cloudron 1162 Jan  3 14:39 utils.js
```

**Comment by AileenCGN** [comment]
I can confirm this issue! When installing with Ghost CLI, the templates are missing in the current npm package of Ghost 1.19.1.

**Comment by kirrg001** [comment]
Was merged and will be released in a bit (1.19.2).
